File: app/src/main/java/com/slutsenko/cocktailapp/CocktailList.java
package com.slutsenko.cocktailapp;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

class CocktailList {

    @SerializedName("drinks")
    @Expose
    private ArrayList<Cocktail> drinks = new ArrayList<Cocktail>();

    ArrayList<Cocktail> getCocktails() {
        return drinks;
    }
}

File: app/src/main/java/com/slutsenko/cocktailapp/data/network/CocktailList.kt
package com.slutsenko.cocktailapp.data.network

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.util.*

class CocktailList {
    @SerializedName("drinks")
    @Expose
    val cocktails = ArrayList<CocktailNetModel>()
}